napari/_vispy/vispy_base_layer.py

```diff
@@ -127,21 +127,25 @@ def _on_blending_change(self, event=None):
         self.node.update()
 
     def _on_scale_change(self, event=None):
-        self.scale = [
-            self.layer.scale[d] * self.layer._scale_view[d]
-            for d in self.layer.dims.displayed[::-1]
-        ]
+        scale = (
+            self.layer._transform_view.compose(self.layer._transform)
+            .set_slice(self.layer.dims.displayed)
+            .scale
+        )
```

Comment:
I don't know if I was ever clear what scale view and scale were, but I'm definitely not clear now. =P Could you clarify in a comment? Are there maybe better names for these?

Reply:
yeah - the whole `grid` / `view` / normal thing is a bit ugly - i will try and improve in the next PR that actually adds the world coordinates. For now I've added a comment directly on the base layer where those are created

Reply:
@sofroniewn thanks! I think it's worth adding a comment here also, as it might not occur to someone reading this code to go find an explanation in the base layer.